Kevin/create controller dir

kevinrosarioc opened the pull request:
In this branch I initialized the controller/todo directory with new controller.js and router.js (each one of them with their needed imports and placeholder functions for passing linting tests in Circle CI). I also added a new todo.controller.tests (also with it's imports). 
There were changes in common/api.yml in order to replace Example placeholders with To Do ones. I also modified the create endpoint to provide a better example.
server/routes.js was also modified to provide route for api/todo instead of the previous example. 
The example test module in the test directory was commented out entirely to pass tests in Circle CI

fernandogruning commented on backend/test/todo.controller.js:
```diff
@@ -0,0 +1,9 @@
+
+const expect = chai.expect;
+
+console.log(
```
Try to at least add a stub test, where you test that the server and the new controller work.

kevinrosarioc commented:
I updated the branch to satisfy requested changes. 
I added a stub method in the todo/controller app and it's corresponding unit test in test/todo.controller.js. I also had to update the todo/router.js module to include a new path for the stub method . There were more changes in api.yml to include the stub function in swagger.

fernandogruning reviewed (approved):
Very good!
